index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/89d87f2112.js" crossorigin="anonymous"></script>
+  <title>Portafolio</title>
+</head>
+
+<body>
+  <header class="header">
+    <div class="tallbar"></div>
+    <div class="header-sub">
+      <h2 class="logo">My logo</h2>
+      <nav>
+        <i class="fa-solid fa-bars"></i>
+      </nav>
+    </div>
+
+  </header>
+
+  <main class="main">
+    <section class="header-section">
+      <div class="title">
+        <h1>I’m TATOU.
+          Glad to see you!</h1>
+      </div>
+      <div class="parrafo">
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work
+          and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+      </div>
+  
+    <div class="connect">
+      <img src="img/LET’S CONNECT.png" alt="connection image">
+    </div>
+    <div class="socialMedia">
+      <a href=""><img src="img/Vector.png" alt="" class="icon"></a>
+      <a href=""><img src="img/Vector (1).png" alt="" class="icon"></a>
+      <a href=""><img src="img/Vector (2).png" alt="" class="icon icon-3"></a>
+      <a href=""><img src="img/Vector (3).png" alt="" class="icon"></a>
+      <a href=""><img src="img/Vector (4).png" alt="" class="icon"></a>
+    </div>
```

Comment:
You did well fixing the background illustration but you forgot two other issues raised by the previous review.

- The social media icons should use a `ul` as the container and each icon wrapped with a `li`. See the [review here](https://github.com/tatouroberto/portfolio-project/pull/1#discussion_r1181103069). 
- The logo should be wrapped with an `<a>` tag. See review [comment here](https://github.com/tatouroberto/portfolio-project/pull/1#discussion_r1181098996).

Kindly fix the above issues.